From: Steve French <stfrench@microsoft.com>
Date: Mon, 24 Jun 2019 01:19:52 -0500
Subject: [PATCH] cifs: Fix check for matching with existing mount
Git-commit: 6552d6a0264de505d3bb1445aecf757b1ec3873a
Patch-mainline: v5.3-rc1
References: bsc#1144333

If we mount the same share twice, we check the flags to see if the
second mount matches the earlier mount, but we left some flags out.

Signed-off-by: Steve French <stfrench@microsoft.com>
Acked-by: Paulo Alcantara <palcantara@suse.de>
---
 fs/cifs/cifsglob.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fs/cifs/cifsglob.h b/fs/cifs/cifsglob.h
index 4777b3c4a92c..88c98fa15f39 100644
--- a/fs/cifs/cifsglob.h
+++ b/fs/cifs/cifsglob.h
@@ -617,6 +617,7 @@ struct smb_vol {
 			 CIFS_MOUNT_FSCACHE | CIFS_MOUNT_MF_SYMLINKS | \
 			 CIFS_MOUNT_MULTIUSER | CIFS_MOUNT_STRICT_IO | \
 			 CIFS_MOUNT_CIFS_BACKUPUID | CIFS_MOUNT_CIFS_BACKUPGID | \
+			 CIFS_MOUNT_UID_FROM_ACL | CIFS_MOUNT_NO_HANDLE_CACHE | \
 			 CIFS_MOUNT_NO_DFS)
 
 /**
-- 
2.22.0

